util/include/OpenTracing.hpp

```diff
@@ -21,7 +21,15 @@
 
 namespace concordUtils {
 
-using SpanContext = std::string;
+class SpanContext {
+ public:
+  SpanContext() = default;
+  explicit SpanContext(const std::string& data) : data_{data} {}
+  std::string data() const { return data_; }
```

Comment:
Maybe return a reference here, rather than a copy?

Reply:
Good suggestion!
If a developer uses `auto s = ctx.data()` then `auto` won't be a reference type.
Thanks!